accord-core/src/test/java/accord/verify/StrictSerializabilityVerifier.java

```diff
@@ -562,9 +573,15 @@ public void checkForUnwitnessed(int start)
             this.futureWrites = new FutureWrites(key, keyCount);
         }
 
+        void print()
+        {
+            for (Map.Entry<Integer, UnknownStepHolder> e : futureWrites.byWriteValue.entrySet())
+                logger.error("{}: {} -> ({}, {}, {})", key, e.getKey(), e.getValue().writeValue, e.getValue().start, e.getValue().end);
+        }
+
         private void updateSequence(int[] sequence, int maybeWrite)
         {
-            for (int i = 0, max = Math.min(sequence.length, this.sequence.length) ; i < max ; ++i)
+            for (int i = 0, max = Math.min(sequence.length, this.sequence.length); i < max; ++i)
```

Comment:
more unnecessary reformatting

Reply:
Our style guide does not cover this (see https://cassandra.apache.org/_/development/code_style.html) and instead delegates to Oracle's (see http://java.sun.com/docs/codeconv/html/CodeConvTOC.doc.html).  Based off Oracle's style guide, we should not have the spaces

https://www.oracle.com/java/technologies/javase/codeconventions-statements.html#454